dynamic-demo-plugin/src/components/ProxyService.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import { consoleFetchJSON } from '@openshift-console/dynamic-plugin-sdk';
+
+const ProxyService: React.FC = () => {
+  return (
+    <div>
```

Comment:
nit: Not a huge deal, but it would be better to use the PF page layout components like we do for other pages.